microservices/case-service/src/main/java/org/entando/bundle/entity/Process.java

```diff
@@ -31,7 +36,14 @@ public class Process {
   @Column(name = "created", columnDefinition = "TIMESTAMP")
   private LocalDateTime created;
 
-  @Column(nullable = false)
-  private String note;
+  @Column(nullable = false, length = 20)
+  private String identifier;
+
+  @Column(nullable = false, length = 40)
+  private String name;
```

Comment:
Is this supposed to contain the username? In that case I would name this author/creator instead

Reply:
Done